Update Supported OS to include Windows Server 2016

wesadams83 opened the pull request:
At the time of Authorship, Windows 10 was the only supported OS to leverage the New-SelfSignedCertificate.  The current New-SelfSignedCertificate documentation indicates the commandlet now works on both Windows 10 and Windows Server 2016.  I have personally tested each of the example commandlets on Windows Server 2016 and each worked without modification.  

This is a significant distinction as it allows certificates to be created on trusted Windows Server 2016 VMs running in Azure.  It is difficult to provision Windows 10 VMs in Azure.  It also allows the possibility to more completely automate the process via PowerShell and minimize the dependency on MakeCert.

azurecla commented:
Hi __@scottrille__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (scottcot). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

wesadams83 committed:
Update Supported OS to include Windows Server 2016

At the time of Authorship, Windows 10 was the only supported OS to leverage the New-SelfSignedCertificate.  The current New-SelfSignedCertificate documentation indicates the commandlet now works on both Windows 10 and Windows Server 2016.  I have personally tested each of the example commandlets on Windows Server 2016 and each worked without modification.  

This is a significant distinction as it allows certificates to be created on trusted Windows Server 2016 VMs running in Azure.  It is difficult to provision Windows 10 VMs in Azure.  It also allows the possibility to more completely automate the process via PowerShell and minimize the dependency on MakeCert.
